File: app/src/main/java/com/example/myneighborly/SplashActivity.java
package com.example.myneighborly;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;


public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        Button startButton = findViewById(R.id.buttonStart);
        startButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(SplashActivity.this, SignUpActivity.class);
                startActivity(intent);
                finish(); // Stänger splash-skärmen så att den inte kan backas till
            }
        });
    }
}

File: app/src/main/java/com/example/myneighborly/SplashActivity.kt
package com.example.myneighborly

import android.content.Intent
import android.os.Bundle
import android.util.Log
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import com.google.firebase.auth.FirebaseAuth

class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        val startButton: Button = findViewById(R.id.buttonStart)

        startButton.setOnClickListener {
            Log.d("SplashActivity", "Start button clicked")

            val currentUser = FirebaseAuth.getInstance().currentUser
/*
            if (currentUser != null) {
                val intent = Intent(this, MainActivity::class.java)
                startActivity(intent)
                finish()
            } else {



 */
                val intent = Intent(this, SignUpActivity::class.java)
                startActivity(intent)
                finish()
            //}
        }
    }
}
